testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -566,49 +537,48 @@ def _single_frame(self, filename):
 
 
 class TestXTCWriterSingleFrame(_GromacsWriterIssue101):
-    __test__ = True
     ext = ".xtc"
     prec = 2
 
 
 class TestTRRWriterSingleFrame(_GromacsWriterIssue101):
-    __test__ = True
     ext = ".trr"
 
 
-class _GromacsWriterIssue117(TestCase):
-    __test__ = False
+
+
+
+
+
+
+
+
+
+class _GromacsWriterIssue117(object):
     """Issue 117: Cannot write XTC or TRR from AMBER NCDF"""
     ext = None
     prec = 5
 
-    def setUp(self):
+    @pytest.fixture()
+    def universe(self):
         pytest.importorskip('netCDF4')
```

Comment:
Please remove `pytest.importorskip('netCDF4')` - not needed anymore since #503. Thanks.